build-tools/packages/build-cli/src/library/changesets.ts

```diff
@@ -5,22 +5,31 @@
 
 import path from "node:path";
 import { VersionBumpType } from "@fluid-tools/version-tools";
-import { Logger } from "@fluidframework/build-tools";
+import { Logger, type ReleaseNotesSection } from "@fluidframework/build-tools";
 import { compareAsc, formatISO, parseISO } from "date-fns";
 import globby from "globby";
 import matter from "gray-matter";
+const { test: hasFrontMatter } = matter;
 
+import { readFile } from "node:fs/promises";
 import { ReleasePackage } from "../releaseGroups.js";
 import { Repository } from "./git.js";
 
 export const DEFAULT_CHANGESET_PATH = ".changeset";
 
+export interface FluidCustomChangesetMetadata {
+	section?: ReleaseNotesSection["name"];
+	includeInReleaseNotes?: boolean;
+}
```

Comment:
Add docs. This is the type for the additional metadata that can be added to individual changesets.